.github/actions/npm_scripts/action.yml

```diff
@@ -1,14 +1,12 @@
-name: smoke_tests
+name: npm_scripts
```

Comment:
Renaming this CI action to be clearer that we're actually smoke testing the _npm scripts_ (and not retail-react-app site).